app/resources/view.html.template

```diff
@@ -17,15 +17,22 @@
   {{person.full_name}} - {{block.super}}
 {% endblock %}
 
-{% if params.query or params.given_name or params.family_name %}
-  {% block logo %}
+{% block logo %}
+  {% if params.query_name or params.given_name or params.family_name %}
```

Comment:
I think the reason is that "blocks are included regardless of any conditionals around them"
I found the answer from this page:
http://stackoverflow.com/questions/12088222/calling-block-inside-an-if-condition-django-template